pkgs/development/python-modules/comet-ml/default.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  dulwich,
+  everett,
+  importlib-metadata,
+  jsonschema,
+  numpy,
+  psutil,
+  python-box,
+  requests,
+  requests-toolbelt,
+  rich,
+  semantic-version,
+  sentry-sdk,
+  setuptools,
+  simplejson,
+  urllib3,
+  wrapt,
+  wurlitzer,
+}:
+
+buildPythonPackage rec {
+  pname = "comet-ml";
+  version = "3.49.11";
```

```suggestion
  version = "3.49.11";
  pyproject = true;
```
and also add build-system